solidity/contracts/liquidity-protection/LiquidityProtectionSettings.sol

```diff
@@ -351,12 +339,7 @@ contract LiquidityProtectionSettings is ILiquidityProtectionSettings, AccessCont
      *
      * @param subscriber    subscriber address
      */
-    function removeSubscriber(ILiquidityProtectionEventsSubscriber subscriber)
-        external
-        onlyOwner
-        validAddress(address(subscriber))
-        notThis(address(subscriber))
-    {
+    function removeSubscriber(ILiquidityProtectionEventsSubscriber subscriber) external onlyOwner {
```

Comment:
No need to check, because the function will revert anyway if `subscriber` does not point to a valid ILiquidityProtectionEventsSubscriber instance?

Reply:
Kind of. `_subscribers.remove` will revert with any address that wasn't previously verified via `addSubscriber`, which does check this.